src/Services/InteractsWithPage.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace Modelizer\Selenium\Services;
+
+trait InteractsWithPage
```

Comment:
I think this should be singular `InteractWithPage` this will handle only one page at a time and as far as I know we should keep classes, traits, interfaces singular.

What do you think?